ddtrace/contrib/celery/app.py

```diff
@@ -30,6 +30,9 @@ def patch_app(app, pin=None):
         # Patch method
         setattr(app, method_name, wrapt.FunctionWrapper(method, wrapper))
 
+    # patch the Task class if available
+    setattr(app, 'Task', patch_task(app.Task))
```

Comment:
when we patch the app, we may consider to patch also the `Task` otherwise https://github.com/DataDog/dd-trace-py/pull/196/files#diff-10c6d560aac930440a75c594cd2583a7R169 will fail.